packages/linter/src/migrations/update-15-0-0/add-eslint-inputs.spec.ts

```diff
@@ -6,135 +6,193 @@ import {
           },
-          "lint2": Object {
-            "inputs": Array [
-              "default",
-              "{workspaceRoot}/.eslintrc.json",
-            ],
+          lint2: {
+            executor: '@nrwl/linter:eslint',
+            options: {},
+          },
+          notTest: {
+            executor: 'nx:run-commands',
           },
         },
-        "version": 2,
+      });
+    });
+
+    test.each(eslintConfigFileWhitelist)(
+      'should ignore %p for production',
+      async (eslintConfigFilename) => {
+        tree.write(eslintConfigFilename, '{}');
+
+        await addEslintInputs(tree);
+
+        const updated = readWorkspaceConfiguration(tree);
+
+        expect(updated.namedInputs.production).toEqual([
+          'default',
+          `!{projectRoot}/${eslintConfigFilename}`,
+        ]);
       }
-    `);
+    );
+
+    test.each(eslintConfigFileWhitelist)(
+      'should add %p to all lint targets',
+      async (eslintConfigFilename) => {
+        tree.write(eslintConfigFilename, '{}');
+
+        await addEslintInputs(tree);
+
+        const updated = readWorkspaceConfiguration(tree);
+        const result = ['default', `{workspaceRoot}/${eslintConfigFilename}`];
+
+        expect(updated.targetDefaults.lint.inputs).toEqual(result);
+        expect(updated.targetDefaults.lint2.inputs).toEqual(result);
+      }
+    );
   });
 
-  it('should add inputs configuration for .eslintrc.js', async () => {
-    updateWorkspaceConfiguration(tree, {
-      version: 2,
-      namedInputs: {
-        default: ['{projectRoot}/**/*', 'sharedGlobals'],
-        sharedGlobals: [],
-        production: ['default'],
-      },
-    });
-    addProjectConfiguration(tree, 'proj', {
-      root: 'proj',
-      targets: {
-        lint: {
-          executor: '@nrwl/linter:eslint',
-          options: {},
-        },
-        lint2: {
-          executor: '@nrwl/linter:eslint',
-          options: {},
-        },
-        notTest: {
-          executor: 'nx:run-commands',
+  describe('development', () => {
+    beforeEach(() => {
+      tree = createTreeWithEmptyWorkspace();
+
+      addProjectConfiguration(tree, 'proj', {
+        root: 'proj',
+        targets: {
+          lint: {
+            executor: '@nrwl/linter:eslint',
+            options: {},
+          },
+          lint2: {
+            executor: '@nrwl/linter:eslint',
+            options: {},
+          },
+          notTest: {
+            executor: 'nx:run-commands',
+          },
         },
-      },
+      });
     });
-    tree.write('.eslintrc.js', '');
-
-    await addEslintInputs(tree);
-
-    const updated = readWorkspaceConfiguration(tree);
-    expect(updated).toMatchInlineSnapshot(`
-      Object {
-        "namedInputs": Object {
-          "default": Array [
-            "{projectRoot}/**/*",
-            "sharedGlobals",
-          ],
-          "production": Array [
-            "default",
-            "!{projectRoot}/.eslintrc.json",
```

Comment:
@FrozenPandaz This is the snapshot in question.

Reply:
Probably a typo. Interestingly how that worked.